### PR TITLE
Fixed typos, images, added missed setting on customer engagement page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Fixed typos, images, added missed setting on customer engagement page

## Affected documentation pages

https://docs.magento.com/user-guide/reports/customer-engagement.html

### CODE REVIEW

File: src/reports/customer-engagement.md [hunk 1]
```diff
@@ -165,5 +165,12 @@ If you leave this field empty, a security check does not run. We do not recommen
...
\ No newline at end of file
+   ![System Alerts]({% link reports/assets/system.alerts.block.png %}){: .zoom}
+   _System Alerts_
+
+## PWA Settings
+
+Specify **PWA Storefront Base URL** field if you use a PWA storefront URL which is different from Magento's default frontend base URL. e.g. _https://pwa.yourdomain.com/_.
+
+   ![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
+   _PWA Base URL_
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:176]:
```suggestion
## PWA settings

If you use a PWA storefront URL, specify the **PWA Storefront Base URL** (for example, `https://pwa.yourdomain.com/`). This is different from the default store frontend base URL.

![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
_PWA storefront base URL_
```

**Review by leslietilling — COMMENTED** [review]
@DmitryTsymbal 

All of the dotdigital content will be removed with the 2.4.4 release. See: https://github.com/magento/merchdocs/commit/6d0f181229a82b3e1881688b3282a286a791dceb

We can make these fixes in the meantime, but I would prefer not to add a new image file that will have to be cleaned up later.

File: src/reports/customer-engagement.md (hunk 1 above)
Comment (DmitryTsymbal) [reply src/reports/customer-engagement.md]: Hello @leslietilling , thank you for the review and noticing. I have commited suggested changes and also deleted the previously added image.
Thanks!

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth an error occurred during the Pull Request import.

**Pull request closed without merging**